src/std/misc/process.ss

```diff
@@ -4,10 +4,11 @@
 
 (export
   run-process
-  run-process/batch)
+  run-process/batch
+  filter-with-process)
 
 (import
-  :gerbil/gambit/ports
+  :gerbil/gambit/ports :gerbil/gambit/threads
   :std/misc/ports :std/sugar)
```

Comment:
Can we tidy these imports at one per line? I want to be able to quickly see what the dependencies are, without straining my eyes.